int8                       type 
int32                      id
geometry_msgs/PoseStamped  pose
